src/components/AccountSecurity/IssueReport.tsx

```diff
@@ -157,9 +157,12 @@ class IssueReport extends Component<Props, State, {}> {
   handleSubmitWithRecaptcha = async (event: any) => {
     event.preventDefault();
     if (recaptchaRef !== null && recaptchaRef.current !== null) {
+      // @ts-ignore
```

Comment:
What's this and why is it commented out?

Reply:
Without it it gives me a typescript error